SenseGlove/README.md

```diff
@@ -0,0 +1,57 @@
+## HapticGlove
+
+Here we are exposing the [SenseGlove SDK](https://github.com/Adjuvo/SenseGlove-API) as a wearable device. This device expose the interfaces in order to stream hand motion data and setting the haptic feedback to the glove through the YARP netwrok.
+
+The following interfaces are exposed for each hand/glove:
+
+- human hand joint angles (computed by SenseGlove SDK) `[20 DoF (each finger 4 DoF), rad]`
+- glove fingertip poses attached to the human fingertip wrt human frame `5 pose vectors [<position, quaternion>]`
+- glove hand pose (using the hand IMU data) wrt human hand inertial frame`Pose [<position, quaternion>]`
+- hand fingertip vibrotactile feedbacks `[5 values: 0.0 - 1.0]`
+- hand fingertip forceFeedback feedbacks `[5 values: 0.0 - 1.0]`
+- hand palm vibrotactile feedback `[1 value: 0.0 - 1.0]`
+
+
+### Dependencies
+
+- Sense Glove SDK
+```
+git clone https://github.com/Adjuvo/SenseGlove-API
+cd SenseGlove-API
+git checkout v2.102.1
+```
+In a Linux machine add following environment variable:
```

Comment:
simple typo: In a Linux machine add the following environment variables:

Reply:
Fixed with https://github.com/ami-iit/yarp-device-haptic-gloves/pull/3/commits/86bf5c2a5f7a25a8a72a064cbfd1a7d67485311c